### ISSUE TITLE
errors when submitting project

### ISSUE THREAD

**Issue by afshinjafari** [cyrillef/extract.autodesk.io#6]
hi,
with the new build I get this error when submitting the project:

**C:\Temp\...>node start.js
Server listening on port 8083
Token: 4zKnRKJzYLFzLYNNNEndtdhmv2QR
Token RO: Px84bQovzVjQMgCsb3zOg1lMy4hm
POST done beam.nwc 632486-beamnwc
{ statusCode: 403, statusMessage: 'Forbidden' }**

and with another model:
**POST done KLE_DomGasPart.NWD 113114670-KLE_DomGasPartNWD
POST done KLE_DomGasPart.NWD 113114670-KLE_DomGasPartNWD
Unexpected end of JSON input 113114670-KLE_DomGasPartNWD
Unexpected end of JSON input 113114670-KLE_DomGasPartNWD
{ Error: ETIMEDOUT
    at Timeout._onTimeout (C:\Temp\extract.autodesk.io-master\node_modules\reque
st\request.js:759:15)
    at tryOnTimeout (timers.js:224:11)
    at Timer.listOnTimeout (timers.js:198:5) code: 'ETIMEDOUT', connect: false }

Unexpected end of JSON input 113114670-KLE_DomGasPartNWD
Unexpected end of JSON input 113114670-KLE_DomGasPartNWD**

and stops uploading the model, nothing will happen next and cannot browse the model.
any ideas?

**Comment by cyrillef** [comment]
can you share the model for me to try, or at least tell me the file size?

**Comment by cyrillef** [comment]
Oh, no stupid me - I know what the issue is... I'll make a fix - that was something I had to do on my list but has not made it in yet. will do a fix asap

**Comment by cyrillef** [comment]
tested and working, let me know if any problem

**Comment by afshinjafari** [comment]
Hi Cyrille,
Thanks for the update,
Will test on Monday (will be your Sunday evening as I'm in Australia EST) and update you.

thanks again and have a great weekend,
Cheers,
Afshin

**Comment by afshinjafari** [comment]
hi Cyrille,

sorry to tell that I still have the issue with the latest build:
(clean copy and npm install)

**extract-autodesk-io-2017 does not exist? { statusCode: 403, statusMessage: 'Forb
idden' }
POST partly_done KLE_DomGasPart.NWD 113114670-KLE_DomGasPartNWD
POST done KLE_DomGasPart.NWD 113114670-KLE_DomGasPartNWD
POST done KLE_DomGasPart.NWD 113114670-KLE_DomGasPartNWD
{ Error: ETIMEDOUT
    at Timeout._onTimeout (C:\Temp\extract.autodesk.io-master\node_modules\forge
-apis\node_modules\request\request.js:759:15)
    at tryOnTimeout (timers.js:224:11)
    at Timer.listOnTimeout (timers.js:198:5) code: 'ETIMEDOUT', connect: false }**

testing using a smaller file:
**POST done beam.nwc 632486-beamnwc
{ statusCode: 403, statusMessage: 'Forbidden' }**

first file is about 100 MB and the second file is only 600 KB
any thoughts?
Cheers,
Afshin

**Comment by afshinjafari** [comment]
I think the issue is from bellow code I see you have these two lines of code:

**if ( md5sum !== '37e06274b75aa068cf2d40af85248953' )
	config.bucket +=config.client_id.toLowerCase () ;**

and I believe it should be:

**if ( md5sum == '37e06274b75aa068cf2d40af85248953' )
	config.bucket += config.credentials.client_id.toLowerCase();**

fixing this let the server to create the bucket:

**C:\Temp\extract.autodesk.io-master>node start.js
Server listening on port 80
Token RO: HWm.....
Token: LwxCu1.....
extract-autodesk-io-2017............... does not exist? { statusCode: 404, statusMessage: 'Not Found' }
Creating bucket:  extract-autodesk-io-2017...............
extract-autodesk-io-2017............... created! -**

however still getting this error when submitting the project:

**POST done KLE_DomGasPart.NWD 113114670-KLE_DomGasPartNWD
Unexpected end of JSON input 113114670-KLE_DomGasPartNWD
Unexpected end of JSON input 113114670-KLE_DomGasPartNWD
Unexpected end of JSON input 113114670-KLE_DomGasPartNWD
{ Error: ETIMEDOUT
    at Timeout._onTimeout (C:\Temp\extract.autodesk.io-master\node_modules\forge
-apis\node_modules\request\request.js:759:15)
    at tryOnTimeout (timers.js:224:11)
    at Timer.listOnTimeout (timers.js:198:5) code: 'ETIMEDOUT', connect: false }**

which I believe is due to the file size. I tried with a small file (600KB) and all went fine and could download the zip file, however with the larger file (110MB) I got the timeout. Changed the timeout in 'forge-token.js' and now able to process the large model.
now trying to actually show the model in the browser
